atom/browser/native_window_views_win.cc

```diff
@@ -92,15 +92,25 @@ bool NativeWindowViews::PreHandleMSG(
     // accessibility object.
     case WM_GETOBJECT: {
       const DWORD obj_id = static_cast<DWORD>(l_param);
-      if (enabled_a11y_support_) return false;
-
-      if (obj_id == OBJID_CLIENT) {
-        const auto axState = content::BrowserAccessibilityState::GetInstance();
-        if (axState && !axState->IsAccessibleBrowser()) {
-          axState->OnScreenReaderDetected();
-          enabled_a11y_support_ = true;
-          Browser::Get()->OnAccessibilitySupportChanged();
-        }
+      if (checked_for_a11y_support_) return false;
+
+      checked_for_a11y_support_ = true;
```

Comment:
On this branch (compared with master), I'm not seeing the `'accessibility-support-changed'` event fire when you launch NVDA after launching your app:

To reproduce:

- Launch Electron
- Open the dev tools and run the following
```js
require('electron').remote.app.on('accessibility-support-changed', function(e, axs) {console.log('changed', axs)})
```
- Launch NVDA
- 'changed true` is not logged on this branch (it is logged on master)

Should the `checked_for_a11y_support_ = true;` be done after the `if (obj_id != OBJID_CLIENT) {` check? If I move it below that check, then I do see it fire.